docs/source/technical_tutorials/visualization/stac_ipyleaflet.ipynb

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
<p>this can be done inside a notebook code cell with the <code>!</code> (minor not blocking)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
In a code block
```
%pip install -q git+https://github.com/MAAP-Project/stac_ipyleaflet.git#egg-info=stac_ipyleaflet
```